app/lib/pages/chat/page.dart

```diff
@@ -302,10 +328,11 @@ class ChatPageState extends State<ChatPage> with AutomaticKeepAliveClientMixin {
                                 ),
                         ),
                       ),
-                      maxLines: 8,
-                      minLines: 1,
+                      maxLines: null, // Allows unlimited lines
```

Comment:
This was already done by Joan a few days ago
https://github.com/BasedHardware/omi/commit/21bf005a874a51b97246b46441528b041be1ff00

It isn't ideal to have unlimited lines as it isn't a good UX and it would break the UI after certain number of lines